scripts/install-worker.sh

```diff
@@ -194,22 +201,21 @@ for binary in ${BINARIES[*]} ; do
 done
 sudo rm *.sha256
 
-KUBELET_CONFIG=""
 KUBERNETES_MINOR_VERSION=${KUBERNETES_VERSION%.*}
-if [ "$KUBERNETES_MINOR_VERSION" = "1.10" ] || [ "$KUBERNETES_MINOR_VERSION" = "1.11" ]; then
-    KUBELET_CONFIG=kubelet-config.json
-else
-    # For newer versions use this config to fix https://github.com/kubernetes/kubernetes/issues/74412.
-    KUBELET_CONFIG=kubelet-config-with-secret-polling.json
-fi
 
 sudo mkdir -p /etc/kubernetes/kubelet
 sudo mkdir -p /etc/systemd/system/kubelet.service.d
 sudo mv $TEMPLATE_DIR/kubelet-kubeconfig /var/lib/kubelet/kubeconfig
 sudo chown root:root /var/lib/kubelet/kubeconfig
-sudo mv $TEMPLATE_DIR/kubelet.service /etc/systemd/system/kubelet.service
+
+if [ "$KUBERNETES_MINOR_VERSION" = "1.14" ]; then
```

Comment:
I'm not sure where this logic is coming from; its not in the upstream master  anywhere: https://github.com/awslabs/amazon-eks-ami/blob/master/scripts/install-worker.sh ; you might want to make sure that you are pulling in the changes from the latest upstream master

Reply:
removed 1.14 patch